exporter/redisPwd_test.go

```diff
@@ -0,0 +1,18 @@
+package exporter
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestLoadPwdFile(*testing.T) {
+	confFile := "../pwd.json"
+	err := LoadPwdFile(confFile)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	fmt.Println(len(RedisPwd))
```

Comment:
Instead of the Println() - can you please try checking the actual values of the map that's returned from loadPwdFile() and fail if they're not present?